scripts/training/prepare-config.py

```diff
@@ -86,3 +86,91 @@
     yaml.dump(config, yaml_file, default_flow_style=False, default_style='\'', explicit_start=True)
 
 s3_client.upload_file(Bucket=s3_bucket, Key=yaml_key, Filename=local_yaml_path)
+
+
+# Training with different configurations on each worker (aka Multi Config training)
+config['MULTI_CONFIG'] = os.environ.get('DR_MULTI_CONFIG', 'False')
+
+if config['MULTI_CONFIG'] == "True":
+    num_workers = int(os.environ.get('DR_WORKERS',1))
+
+    for i in range(1,num_workers+1,1):
+        if i == 1:
+            # copy training_params to training_params_1
+            s3_yaml_name_list = s3_yaml_name.split('.')
+            s3_yaml_name_temp = s3_yaml_name_list[0] + "_%d.yaml" % i
+
+            #upload additional training params files
+            yaml_key = os.path.normpath(os.path.join(s3_prefix, s3_yaml_name_temp))
+            s3_client.upload_file(Bucket=s3_bucket, Key=yaml_key, Filename=local_yaml_path)
+
+            file1 = open("multiconfig.txt", "w")
```

Comment:
File should be stored in the DR_DIR/tmp directory -- preferrably prefixed with the DR_RUN_ID not to break multiple runs in parallel.

Reply:
Moved file to tmp as requested, but have not been able to figure out right syntax to include DR_RUN_ID.

Reply:
this is now done